examples/authentication.py

```diff
@@ -82,3 +82,49 @@ def token_gen_call(username, password):
     if mockpassword == password and mockusername == username: # This is an example. Don't do that.
         return {"token" : jwt.encode({'user': username, 'data': 'mydata'}, secret_key, algorithm='HS256')}
     return 'Invalid username and/or password for user: {0}'.format(username)
+
+# JWT AUTH EXAMPLE #
+replace_this = False # Replace this placeholder in your implementation.
+config = {
+    'jwt_secret': 'super-secret-key-please-change',
+    # Token will expire in 3600 seconds if it is not refreshed and the user will be required to log in again.
+    'token_expiration_seconds': 3600,
+    # If a request is made at a time less than 1000 seconds before expiry, a new jwt is sent in the response header.
+    'token_refresh_seconds': 1000 
+}
+# Enable authenticated endpoints, example @authenticated.get('/users/me').
+authenticated = hug.http(requires=hug.authentication.json_web_token(hug.authentication.verify_jwt, config['jwt_secret']))
+
+# Check the token and issue a new one if it is about to expire (within token_refresh_seconds from expiry).
```

Comment:
Normally there will be 2 new lines between function and classes and one new line between class methods. Please add a new line.